clang/docs/ClangFormat.rst

```diff
@@ -33,7 +33,7 @@ to format C/C++/Java/JavaScript/JSON/Objective-C/Protobuf/C# code.
   Clang-format options:
 
     --Werror                       - If set, changes formatting warnings to errors
-    --Wno-error=<value>            - If set, don't error out on the specified warning type.
+    --Wno-error=<value>            - If set don't error out on the specified warning type.
```

Comment:
This does not match the content of `ClangFormat.cpp`. I don't know why, but you either fix it manually or check why this is not set correctly.

Reply:
I agree, that's why I did not commit it in the first place. I'll try manually!